### PR TITLE
3.0 preview dev

### PR DESCRIPTION
**Pull request by e111077**
LOTS of files.

logic changes in `NeonAnimationRunnerBehavior`'s `_configureAnimations` function which I SWEAR I remember fixing when we migrated this in google3 but for some reason it's not in master?

Some of the demos didn't seem to actually work before so now they do

### CODE REVIEW

File: demo/shared-styles.js [hunk 1]
```diff
@@ -1,58 +1,58 @@
...
+http://polymer.github.io/AUTHORS.txt The complete set of contributors may be
+found at http://polymer.github.io/CONTRIBUTORS.txt Code distributed by Google as
+part of the polymer project is also subject to an additional IP rights grant
+found at http://polymer.github.io/PATENTS.txt
 */
 /*
   FIXME(polymer-modulizer): the above comments were extracted
   from HTML and may be out of place here. Review them and
   then delete this comment!
 */
```
[the remark this answers is not in the document]
Comment (e111077) [reply demo/shared-styles.js]: oop

**Comment by e111077** [comment]
implemented changes

File: neon-animation-runner-behavior.js [hunk 2]
```diff
@@ -146,5 +152,6 @@ export const NeonAnimationRunnerBehaviorImpl = {
   }
 };
 
-/** @polymerBehavior Polymer.NeonAnimationRunnerBehavior */
-export const NeonAnimationRunnerBehavior = [NeonAnimatableBehavior, NeonAnimationRunnerBehaviorImpl];
+/** @polymerBehavior NeonAnimationRunnerBehavior */
```
Comment (bicknellr) [inline neon-animation-runner-behavior.js]: I think this should be `/** @polymerBehavior */`.

File: neon-shared-element-animatable-behavior.js [hunk 3]
```diff
@@ -29,8 +30,6 @@ export const NeonSharedElementAnimatableBehaviorImpl = {
 
 };
 
-/** @polymerBehavior Polymer.NeonSharedElementAnimatableBehavior */
-export const NeonSharedElementAnimatableBehavior = [
-  NeonAnimatableBehavior,
-  NeonSharedElementAnimatableBehaviorImpl
-];
+/** @polymerBehavior NeonSharedElementAnimatableBehavior */
```
Comment (bicknellr) [inline neon-shared-element-animatable-behavior.js]: `/** @polymerBehavior */`

File: neon-shared-element-animation-behavior.js [hunk 4]
```diff
@@ -66,8 +67,6 @@ export const NeonSharedElementAnimationBehaviorImpl = {
 
 };
 
-/** @polymerBehavior Polymer.NeonSharedElementAnimationBehavior */
-export const NeonSharedElementAnimationBehavior = [
-  NeonAnimationBehavior,
-  NeonSharedElementAnimationBehaviorImpl
-];
+/** @polymerBehavior NeonSharedElementAnimationBehavior */
```
Comment (bicknellr) [inline neon-shared-element-animation-behavior.js]: `/** @polymerBehavior */`

File: README.md [hunk 5]
```diff
@@ -1,299 +1,294 @@
...
-animationConfig: {
-  value: function() {
-    return {
-      // run fade-in-animation on this, and the entry animation on this.$.myAnimatable
-      'entry': [
-        {name: 'fade-in-animation', node: this},
-        {animatable: this.$.myAnimatable, type: 'entry'}
-      ]
-    }
+    setTimeout(() => this.playAnimation, 1000);
```
Comment (bicknellr) [inline README.md]: `this.playAnimation` -> `this.playAnimation()`, there's a handful of these in the README.

**Pull request merged**